vscode-smart-contracts/scripts/linux-vscode-smart-contracts.Jenkinsfile

```diff
@@ -0,0 +1,72 @@
+pipeline {
+    agent { label 'jenkins-worker' }
+    environment {
+        VERSION = sh(
+            returnStdout: true,
+            script: '''\
+                # Extract version number if not set as parameter
+                [ -z "$VERSION" ] && VERSION=$(node -p "require('vscode-smart-contracts/package.json').version")
+                echo -n "$VERSION"
+            '''.stripIndent()
+        )
+        CARGO_CONCORDIUM_VERSION = "${VERSION}"
```

Comment:
I am not sure I understand the purpose of this. The VERSION is extracted from package.json, but now set as the CARGO_CONCORDIUM version.

Reply:
Right, this is a bit weird and the only point here was to provide some sort of default for the variable.

Reply:
I'm guessing `CARGO_CONCORDIUM_VERSION` should just be a parameter, or is the version of this plugin suppose to always match the version of `cargo-concordium`

Reply:
I don't think we want to have them match. I guess a more sane default would be to use the version in `cargo-concordium/Cargo.toml`.

Reply:
I have changed it to use the version found in cargo.toml by default